### ISSUE TITLE
v1.1 crashes on log_level: info

### ISSUE THREAD

**Issue by funkypenguin** [cruise-automation/k-rail#31]
Hey guys,

I just tried to upgrade my helm-based installation from v1.0 to v1.1.

The pods crash with:

```
{"level":"fatal","msg":"invalid log level set: \u003cnil\u003e","time":"2019-11-13T03:45:45Z"}
```

The helm chart's `values.yaml` include the following:

```
config:
  log_level: info
```

If I manually edit the configmap that the helm chart creates, and **remove** the `log_level` line, then the pods will run successfully.

If I remove `log_level` from `values.yaml`, the helm chart just inserts a default into the configmap anyway, and we're back to crashlooping.

Thus far, my only workaround has been to downgrade to v1.0

Cheers!
D

**Comment by dustin-decker** [comment]
That's embarassing. I'll have a look.

**Comment by dustin-decker** [comment]
Fixed in https://github.com/cruise-automation/k-rail/pull/33

Not sure how that slipped out. Thanks for the report. I will cut a bug fix release in a few minutes that include that.

**Comment by dustin-decker** [comment]
Released [v1.1.1](https://github.com/cruise-automation/k-rail/releases/tag/v1.1.1) with the fix.
